ElectoralRegisterResidentInformationApi/V1/Controllers/ElectoralRegisterController.cs

```diff
@@ -12,25 +12,24 @@ namespace ElectoralRegisterResidentInformationApi.V1.Controllers
     [ApiVersion("1.0")]
     public class ElectoralRegisterController : BaseController
     {
-        private readonly IGetAllUseCase _getAllUseCase;
+        private readonly IGetAllResidentsUseCase _getAllResidentsUseCase;
         private readonly IGetResidentByIdUseCase _getResidentByIdUseCase;
-        public ElectoralRegisterController(IGetAllUseCase getAllUseCase, IGetResidentByIdUseCase getResidentByIdUseCase)
+        public ElectoralRegisterController(IGetAllResidentsUseCase getAllResidentsUseCase, IGetResidentByIdUseCase getResidentByIdUseCase)
         {
-            _getAllUseCase = getAllUseCase;
+            _getAllResidentsUseCase = getAllResidentsUseCase;
             _getResidentByIdUseCase = getResidentByIdUseCase;
         }
 
-        //TODO: add xml comments containing information that will be included in the auto generated swagger docs (https://github.com/LBHackney-IT/lbh-base-api/wiki/Controllers-and-Response-Objects)
         /// <summary>
-        /// ...
+        /// Searches residents in the electoral registers.
         /// </summary>
-        /// <response code="200">...</response>
+        /// <response code="200">Residents results matching search criteria.</response>
         /// <response code="400">Invalid Query Parameter.</response>
```

Comment:
Do we not need a 404 status code tooo

Reply:
I'm not sure we need `404` for the list endpoint. I could be wrong, but `404` refers to a resource not being found which feels more like a `get by ID` situation than a `Get everything that is available` situation.

Reply:
Yes that's true, it's only needed for ID